routing/endpointregistry_test.go

```diff
@@ -50,20 +54,34 @@ func TestSetAndGetAnotherKey(t *testing.T) {
 }
 
 func TestDoRemovesOldEntries(t *testing.T) {
+	r := NewEndpointRegistry(RegistryOptions{})
+
 	route := &Route{LBEndpoints: []LBEndpoint{
 		{Host: "existing endpoint"},
+		{Host: "removed endpoint"},
```

Comment:
This is not remove yet. Lets call them `endpoint1.test:80` and `endpoint2.test:80`